src/seedu/addressbook/data/person/Person.java

```diff
@@ -14,8 +14,10 @@
     private Phone phone;
     private Email email;
     private Address address;
+    private int sequenceNumber;
 
     private final UniqueTagList tags;
+    private static int nextSequenceNumber = 1;
```

Comment:
This is correct, think why setting it to 0 is incorrect which some students did.